### PR TITLE
docs: Releases WG meeting notes for 5/22

### PR DESCRIPTION
**Pull request by sofianguy**

### CODE REVIEW

File: wg-releases/meeting-notes/2019-05-22.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    * We do not want to support a process model that chromium doesn't support; chromium is constantly changing how that works and it's a pain for us to upkeep
+    * Postponing discussion to next week after Sofia surveys AFP apps about their affinity mode usage
+2. (**@jkleinsc**) Moving releases over to AppVeyor cloud
+    * Two PRs need review:
+        *  https://github.com/electron/electron/pull/18337
+        *  https://github.com/electron/sudowoodo/pull/120
+
+## Backport Requests
+
+1. #18343 ["fix: correctly support the webkitdirectory input attr"](https://github.com/electron/electron/pull/18343) question for Electron **5**, see PR comment discussion. (Added by @ckerr)
```
Comment (codebytere) [inline wg-releases/meeting-notes/2019-05-22.md]: https://github.com/electron/electron/pull/18389 was also closed, the verdict for which should also definitely be in these notes!

Comment (sofianguy) [reply wg-releases/meeting-notes/2019-05-22.md]: It is in backport request #2

Comment (codebytere) [reply wg-releases/meeting-notes/2019-05-22.md]: 🤦‍♀ you are entirely correct, apologies! in that case, would you mind amalgamating them to make that clearer?

Comment (sofianguy) [reply wg-releases/meeting-notes/2019-05-22.md]: updated!

**Pull request merged**